### PR TITLE
[MDB IGNORE] Baytiles/the great ship prettyfication

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Revives baytiles from [here.](https://github.com/Whitesands13/Whitesands/pull/241)
updates a lot of ships to use the new aesthetics
Makes the refrences in the shetland more obvious
image
example

## Why It's Good For The Game
eye candy, seperates aesthetically from a certain abomination  

## Changelog
:cl:
add: baytiles
soundadd: swifterjackie
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
already?

**Comment by triplezeta** [comment]
i wonder why

**Comment by swifterjack** [comment]
rude.

**Comment by swifterjack** [comment]
walls suck give me the cool tiles remove the toblerones

**Comment by rye-rice** [comment]
walls will probably not be in the final version, just in so they make the tiles look good

**Comment by triplezeta** [comment]
walls are based swifter stfu

**Comment by rye-rice** [comment]
It is time
i wont touch the new ships

**Comment by rye-rice** [comment]
@triplezeta if this gets another conflict its going back to brazil

File: code/__DEFINES/colors.dm [hunk 1]
```diff
@@ -8,6 +8,20 @@
 #define COLOR_DARKMODE_BACKGROUND "#272727"
 #define COLOR_DARKMODE_HEADER "#ffffff"
 #define COLOR_DARKMODE_TEXT "#000000"
+
+
+#define COLOR_MONOTILE        	"#545c68"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define COLOR_MONOTILE "#545c68"
```

File: code/__DEFINES/colors.dm [hunk 2]
```diff
@@ -8,6 +8,20 @@
...
+#define COLOR_TILE_GRAY			"#4c535b"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define COLOR_TILE_GRAY "#4c535b"
```

File: code/__DEFINES/colors.dm [hunk 3]
```diff
@@ -8,6 +8,20 @@
...
+#define COLOR_TILE_DARK_GRAY	"#404040"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define COLOR_TILE_DARK_GRAY "#404040"
```

File: code/__DEFINES/colors.dm [hunk 4]
```diff
@@ -8,6 +8,20 @@
...
+#define COLOR_WARNING			"#D2D53D"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define COLOR_WARNING "#D2D53D"
```

File: code/__DEFINES/colors.dm [hunk 5]
```diff
@@ -8,6 +8,20 @@
...
+
+#define WOOD_COLOR_GENERIC     "#d5a66e"
+#define WOOD_COLOR_RICH        "#792f27"
+#define WOOD_COLOR_PALE        "#d2bc9d"
+#define WOOD_COLOR_PALE2       "#e6d2ba"
+#define WOOD_COLOR_BLACK       "#332521"
+#define WOOD_COLOR_CHOCOLATE   "#543c30"
+#define WOOD_COLOR_YELLOW      "#e3994e"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_YELLOW "#e3994e"
```

File: code/__DEFINES/colors.dm [hunk 6]
```diff
@@ -8,6 +8,20 @@
...
 #define COLOR_DARKMODE_HEADER "#ffffff"
 #define COLOR_DARKMODE_TEXT "#000000"
+
+
+#define COLOR_MONOTILE        	"#545c68"
+#define COLOR_TILE_GRAY			"#4c535b"
+#define COLOR_TILE_DARK_GRAY	"#404040"
+#define COLOR_WARNING			"#D2D53D"
+
+#define WOOD_COLOR_GENERIC     "#d5a66e"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_GENERIC "#d5a66e"
```

File: code/__DEFINES/colors.dm [hunk 7]
```diff
@@ -8,6 +8,20 @@
...
+#define WOOD_COLOR_RICH        "#792f27"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_RICH "#792f27"
```

File: code/__DEFINES/colors.dm [hunk 8]
```diff
@@ -8,6 +8,20 @@
...
+#define WOOD_COLOR_PALE        "#d2bc9d"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_PALE "#d2bc9d"
```

File: code/__DEFINES/colors.dm [hunk 9]
```diff
@@ -8,6 +8,20 @@
...
+#define WOOD_COLOR_PALE2       "#e6d2ba"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_PALE2 "#e6d2ba"
```

File: code/__DEFINES/colors.dm [hunk 10]
```diff
@@ -8,6 +8,20 @@
...
+#define WOOD_COLOR_BLACK       "#332521"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_BLACK "#332521"
```

File: code/__DEFINES/colors.dm [hunk 11]
```diff
@@ -8,6 +8,20 @@
...
+#define WOOD_COLOR_CHOCOLATE   "#543c30"
```
Comment (ZephyrTFA) [inline code/__DEFINES/colors.dm]:
```suggestion
#define WOOD_COLOR_CHOCOLATE "#543c30"
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by ZephyrTFA** [comment]
Atmospherics please stop breaking my CI

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: aabc6f1b144d0c36cf9f9f5d49313c6586ec6706
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: 4a612b6abc77d3aaae270b65dc4c94667e4d6be2

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 293bad3e54aa3f26644d5d871baee9eda36057ca
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: 2c17e761a3dbe846412b0eb5224178d8c7dfc1e4

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: d79659920aec58007df63d3e9b96cb0fe1a16353
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: 72d94bef2620899bd315f660ccb547df14ffc328

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: dd854e7386a7a9a664d6357aab6213b032e1817b
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: d097fa6e32a13458e756307a43c1764f99b243bc

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 4d98cdcea7b4591b94a06237309039f4980217fa
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: 9a29cfa82da92d64f18a8f8bcd69d3ea934be1ac

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: f789e74004b3acb82690d77398848b240a153b6f
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: fec71ade6e716fb63d267abca95b86e22f5eb182

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: bf2a3d7daffd8042d3adbe075ec803cd3abd4e03
Pull Request: abd9fd0b092352a4f3b811a7cdc50c3ae98fb2dd
Server: ca0ff8b9f76846976a9d8d2ec5e18cf60ecd28be

**Comment by Sun-Soaked** [comment]
Okay so- it would be a good idea to make these a separate entity from old walls/tiles/etc., and retain them in code, but change construction pathing to the new tiles and walls. Otherwise, you're looking at having to be immediately redoing *every structure, ruin, and ship in the code* which is frankly a reality too terrible to even conceive.

Over time we can shift everything to newtiles, and then kill oldtiles afterwards, but for now I think backwards compat is the way to go.

**Comment by swifterjack** [comment]
btw, this breaks the syndie S 3x3 decal for some odd reason

**Comment by rye-rice** [comment]
this breaks almost every single decal, but i think i can fix some things

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: 033ea7d0a6e5ade2feced249d03f7bfcd3eac439
Pull Request: 7361c3ac159212e93184addae2f7f93d77a274a7
Server: 7c123be6a9d9ffcf192aa21466691c922a728b54

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 9693b436ac8eb8d68349b284df72bb252209581f
Pull Request: 7361c3ac159212e93184addae2f7f93d77a274a7
Server: 651bb7bd5250da841cf57899f6ff939ae8b47202

**Comment by rye-rice** [comment]
🇧🇷

**Comment by ZephyrTFA** [comment]
You are failing integration tests.
Invalid components are being attached to parent objects.
`REASON #6: [03:48:33] Runtime in unsorted.dm,1056: Incompatible /datum/component/decal assigned to a /turf/open/floor/plasteel/mono/white`

**Comment by rye-rice** [comment]
uhhhhhhhhhh @MarkSuckerberg help

**Comment by MarkSuckerberg** [comment]
pain, I will look at it tomorrow

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: ca83a63d9154e7bd47271dd9f5ce5d44eb19d7af
Pull Request: b064af1f906d3d5a7928bd1da86beba2cb2a7da3
Server: b9679cc28f47ef4054511559ac3e5f35dd3100bb

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: ca83a63d9154e7bd47271dd9f5ce5d44eb19d7af
Pull Request: b064af1f906d3d5a7928bd1da86beba2cb2a7da3
Server: 52f18b307da9ed9bafb2f173c5db9806fde13a70

**Comment by ZephyrTFA** [comment]
Check your turf decals. Something has a null icon. 
That is why you are getting this runtime. 
```[03:12:44] Runtime in unsorted.dm,1056: Incompatible /datum/component/decal assigned to a /turf/open/floor/plasteel/mono/white! args: [null,"siding_line",4,0,null,null,null,255]```

**Comment by ZephyrTFA** [comment]
This is good to merge, you just need to fix your null icons so that integration tests succeed

**Comment by MarkSuckerberg** [comment]
Fixed that, it's just atmos runtimes now

**Comment by ZephyrTFA** [comment]
Gonna cry

**Comment by MarkSuckerberg** [comment]
atmos runtimes my beloathed 😭😭😭

**Comment by MarkSuckerberg** [comment]
LGTM, YOLO, LMAO, etcetera

**Comment by rye-rice** [comment]
YES FDKSJLHOIPBHNA MJK<ALDHIUSHBMNLIU<JHBMNKJ S<LIO>:FSF><LKSJO IPF(*DLPM:<SDPKFJMNODF:PI(JUKDLSUNO

**Pull request merged**